vsphere/resource_vsphere_datastore_cluster.go

```diff
@@ -0,0 +1,879 @@
+			},
+			"sdrs_policy_enforcement_automation_level": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Overrides the default automation settings when correcting storage and VM policy violations.",
+				ValidateFunc: validation.StringInSlice(storageDrsPodConfigInfoBehaviorAllowedValues, false),
+			},
+			"sdrs_vm_evacuation_automation_level": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "Overrides the default automation settings when generating recommendations for datastore evacuation.",
+				ValidateFunc: validation.StringInSlice(storageDrsPodConfigInfoBehaviorAllowedValues, false),
+			},
+			"sdrs_io_load_balance_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Enable I/O load balancing for this datastore cluster.",
+			},
+			"sdrs_default_intra_vm_affinity": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "When true, storage DRS keeps VMDKs for individual VMs on the same datastore by default.",
+			},
+			"sdrs_io_latency_threshold": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      15,
+				Description:  "The I/O latency threshold, in milliseconds, that storage DRS uses to make recommendations to move disks from this datastore.",
+				ValidateFunc: validation.IntBetween(5, 100),
+			},
+			"sdrs_io_load_imbalance_threshold": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      5,
+				Description:  "The difference between load in datastores in the cluster before storage DRS makes recommendations to balance the load.",
+				ValidateFunc: validation.IntBetween(1, 100),
+			},
+			"sdrs_io_reservable_iops_threshold": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "The threshold of reservable IOPS of all virtual machines on the datastore before storage DRS makes recommendations to move VMs off of a datastore.",
+			},
+			"sdrs_io_reservable_percent_threshold": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      60,
+				Description:  "The threshold, in percent, of actual estimated performance of the datastore (in IOPS) that storage DRS uses to make recommendations to move VMs off of a datastore when the total reservable IOPS exceeds the threshold.",
+				ValidateFunc: validation.IntBetween(30, 100),
+			},
+			"sdrs_io_reservable_threshold_mode": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      string(types.StorageDrsPodConfigInfoBehaviorAutomated),
+				Description:  "The reservable IOPS threshold to use, percent in the event of automatic, or manual threshold in the event of manual.",
+				ValidateFunc: validation.StringInSlice(storageDrsPodConfigInfoBehaviorAllowedValues, false),
+			},
+			"sdrs_load_balance_interval": {
+				Type:         schema.TypeInt,
```

Comment:
I wonder if it would be better UX to express this as a `string`/`time.Duration`

Reply:
I think this is minor, especially in the face of if we changed it here to something like `60m`/`1h`/`2d`, we would probably need to change it _everywhere_, else we risk harming the UX more than enriching it.